docs/operating/troubleshooting.md

```diff
@@ -5,4 +5,35 @@ menu: operating
 slug: /troubleshooting.md
 ---
 
-Work in progress (:
+# Troubleshooting; Common cases
+
+
+## Overlaps
+
+Overlaps should never happen in healthy and well configured Thanos system. That's why there is no automatic repair.
+
+Let's take two examples:
+
+- `msg="critical error detected; halting" err="compaction failed: compaction: pre compaction overlap check: overlaps found while gathering blocks. [mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s, blocks: 2]: <ulid: 01D94ZRM050JQK6NDYNVBNR6WQ, mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s>, <ulid: 01D8AQXTF2X914S419TYTD4P5B, mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s>`
+
+It shows that you have really the same blocks for 2h overlaps. It worth checking whether are those blocks the same (check the number of series and samples, checksum the index and chunks).
+
+- `msg="critical error detected; halting" err="compaction failed: compaction: pre compaction overlap check: overlaps found while gathering blocks. [mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s, blocks: 2]: <ulid: 01D94ZRM050JQK6NDYNVBNR6WQ, mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s>, <ulid: 01D8AQXTF2X914S419TYTD4P5B, mint: 1555128000000, maxt: 1555142400000, range: 4h0m0s>`
+
+It shows there are two blocks overlapping in 2h time range. One is 4h, the other is 2h.It's more tricky. This suggests you uploaded compacted blocks rather. Again looking on meta.json would help.
```

Comment:
```suggestion
This case is more tricky. It shows there are two blocks overlapping in 2h time range. One is 4h, the other is 2h. This might suggest a manual uploaded of compacted blocks or multiple compactors (which is wrong). Again looking on meta.json would help.
```